mods/cnc/maps/nod05/nod05.lua

```diff
@@ -0,0 +1,319 @@
+NodUnitsVehicle = { 'bike', 'bike', 'bggy', 'ltnk', 'bike', 'bike' }
+NodUnitsRocket = { 'e1', 'e1', 'e1', 'e1' }
+NodUnitsGunner = { 'e3', 'e3', 'e3', 'e3' }
+GDIReinforceUnits = { 'e2', 'e2', 'e2', 'e2', 'e2' }
+
+Gdi1Units = { ['e1'] = 3, ['e2'] = 1 }
+Gdi2Units = { ['e1'] = 2, ['e2'] = 1 }
+Gdi3Units = { ['jeep'] = 1 }
+Gdi4Units = { ['mtnk'] = 1 }
+Gdi5Units = { ['e1'] = 1, ['e2'] = 2 }
+Gdi6Units = { ['e1'] = 3 }
+Gdi7Units = { ['e2'] = 2 }
+Gdi8Units = { ['e2'] = 5 }
+
+AllUnits = { Gdi1Units, Gdi2Units, Gdi3Units, Gdi4Units, Gdi5Units, Gdi6Units, Gdi7Units, Gdi8Units }
+
+XxxxTriggerFunctionTime = DateTime.Minutes(1) + DateTime.Seconds(40)
+YyyyTriggerFunctionTime = DateTime.Minutes(1) + DateTime.Seconds(30)
+ZzzzTriggerFunctionTime = DateTime.Minutes(2) + DateTime.Seconds(30)
+Grd1TriggerFunctionTime = DateTime.Seconds(3)
+Atk2TriggerFunctionTime = DateTime.Minutes(1) + DateTime.Seconds(10)
+Atk3TriggerFunctionTime = DateTime.Minutes(3) + DateTime.Seconds(10)
+Atk4TriggerFunctionTime = DateTime.Minutes(4) + DateTime.Seconds(40)
+Atk6TriggerFunctionTime = DateTime.Minutes(2) + DateTime.Seconds(30)
+
+DelyCellTriggerActivator = { CPos.New(29,30), CPos.New(28,30), CPos.New(27,30), CPos.New(26,30), CPos.New(25,30), CPos.New(24,30), CPos.New(23,30), CPos.New(22,30), CPos.New(21,30), CPos.New(29,29), CPos.New(28,29), CPos.New(27,29), CPos.New(26,29), CPos.New(25,29), CPos.New(24,29), CPos.New(23,29), CPos.New(22,29) }
+DelzCellTriggerActivator = { CPos.New(29,27), CPos.New(28,27), CPos.New(27,27), CPos.New(26,27), CPos.New(25,27), CPos.New(24,27), CPos.New(29,26), CPos.New(28,26), CPos.New(27,26), CPos.New(26,26), CPos.New(25,26), CPos.New(24,26) }
+Atk5CellTriggerActivator = { CPos.New(10,33), CPos.New(9,33), CPos.New(8,33), CPos.New(9,32), CPos.New(8,32), CPos.New(7,32), CPos.New(8,31), CPos.New(7,31), CPos.New(6,31) }
+Atk1CellTriggerActivator = { CPos.New(10,33), CPos.New(9,33), CPos.New(8,33), CPos.New(9,32), CPos.New(8,32), CPos.New(7,32), CPos.New(8,31), CPos.New(7,31), CPos.New(6,31) }
+
+Gdi1Waypoints = { waypoint0, waypoint1, waypoint3, waypoint4 }
+Gdi2Waypoints = { waypoint0, waypoint1, waypoint3, waypoint4, waypoint5 }
+Gdi3Waypoints = { waypoint0, waypoint1, waypoint2 }
+Gdi5Waypoints = { waypoint0, waypoint1, waypoint3, waypoint1, waypoint6 }
+Gdi11Waypoints = { waypoint0, waypoint1, waypoint3, waypoint4, waypoint7, waypoint8 }
+Gdi12Waypoints = { waypoint0, waypoint1, waypoint3, waypoint11, waypoint12 }
+
+AllWaypoints = { Gdi1Waypoints, Gdi2Waypoints, Gdi3Waypoints, Gdi5Waypoints, Gdi11Waypoints, Gdi12Waypoints }
+
+HuntActorTriggerActivator = { Tower1, Tower2, Radar, Silo1, Silo2, Silo3, Refinery, Barracks, Plant1, Plant2, Yard, Factory }
+
+GDIStartUnits = {}
+
+XxxxTriggerFunction = function()
+	if not Radar.IsDead then
+		local list = Nod.GetGroundAttackers()
+		AirstrikeProxy = Actor.Create("airstrike.proxy", true, { Owner = GDI })
```

Comment:
You don't need to recreate and destroy this everytime you want to use it.
Just putting this line somewhere into `WorldLoaded` and removing [line 49](https://github.com/OpenRA/OpenRA/pull/8087/files#diff-dfc35658b310ca620e93326d5670b9adR49) should work.